fvm/evm/stdlib/contract.cdc

```diff
@@ -826,13 +827,17 @@ contract EVM {
         }
     }
 
-    /// createHeartBeat creates a heartbeat resource
+    /// setupHeartbeat creates a heartbeat resource and saves it to storage.
+    /// The function is called once during the contract initialization.
+    ///
+    /// The heartbeat resource is used to control the block production,
+    /// and used in the Flow protocol to call the heartbeat function once per block.
     access(account)
-    fun createHeartBeat(): @Heartbeat{
-        return <-create Heartbeat()
+    fun setupHeartbeat() {
```

Comment:
Refactored, so that we can maybe try call it from the migration.

A manual transaction could also call this, e.g. for Previewnet